packages/ui-testable/src/testable.ts

```diff
@@ -83,32 +84,28 @@ const testable =
           }
           appendLocatorAttribute() {
             this._locatorTimeout = setTimeout(() => {
-              let node: HTMLSpanElement | Element | Text | null = null
+              let node: Element | undefined = undefined
               if (this._testableUnmounted) {
                 return
               }
               try {
                 // Use this.DOMNode for components that render as non-native Portals...
                 node =
-                  (this as ComponentWithDOMNode).DOMNode || findDOMNode(this)
+                  (this as ComponentWithDOMNode).DOMNode ||
+                  (findDOMNode(this) as Element)
```

Comment:
this is the change that removed reactDOM.findDOMNode and uncovered all the fixed errors.